src/Arcus.Testing.Integration.DataFactory/TemporaryDataFlowDebugSession.cs

```diff
@@ -277,12 +277,27 @@ private async Task StartDataFlowAsync(string dataFlowName, RunDataFlowOptions op
         private DataFlowDebugPackageDebugSettings CreateDebugSettings(RunDataFlowOptions options)
         {
             var settings = new DataFlowDebugPackageDebugSettings();
+
             foreach (KeyValuePair<string, BinaryData> parameter in options.DataFlowParameters)
             {
                 _logger.LogTrace("[Test:Setup] Add DataFlow parameter '{Name}' to debug session", parameter.Key);
                 settings.Parameters[parameter.Key] = parameter.Value;
             }
 
+            foreach (KeyValuePair<string, IDictionary<string, object>> datasetParameter in options.DataSetParameters)
+            {
+                foreach (KeyValuePair<string, object> parameter in datasetParameter.Value)
+                {
+                    _logger.LogTrace("[Test:Setup] Add DataSet parameter '{ParameterName}' for DataSet '{DataSetName}' to debug session",
```

Comment:
I would not output the param values just in case it indeed contain sensitive info. Also they are passed by the user, so I trust users to know what they are providing as values.